lib/shared/permissions.js

```diff
@@ -198,6 +198,13 @@ exports.elevateCommand = (command, options) => {
   // for now, we should make sure we double check if the error messages
   // have changed every time we upgrade `sudo-prompt`.
 
+  }).catch((error) => {
+    return _.includes(_.trim(error.message), 'is not in the sudoers file');
```

Comment:
But `_.trim` only removes whitespace from the start and end of a string, and I'd expect `_.includes` to work the same regardless of leading or trailing whitespace?